packages/workbox-precaching/src/precacheAndRoute.ts

```diff
@@ -27,10 +28,11 @@ import './_version.js';
  */
 function precacheAndRoute(
   entries: Array<PrecacheEntry | string>,
-  options?: PrecacheRouteOptions,
+  routeOptions?: PrecacheRouteOptions,
+  controllerOptions?: PrecacheControllerOptions,
```

Comment:
Not a big fan of the two option arguments. Combining them into a single options argument in a non-breaking manner didn't sit well with me either.